python/delta_sharing/tests/test_protocol.py

```diff
@@ -186,11 +186,11 @@ def test_share_profile_bearer(tmp_path):
         DeltaSharingProfile.read_from_file(io.StringIO(json))
 
 
-def test_share_profile_oauth2(tmp_path):
+def oauth_client_credentials(tmp_path):
     json = """
         {
             "shareCredentialsVersion": 2,
-            "type": "persistent_oauth2.0",
+            "type": "oauth_client_credentials",
```

Comment:
will changing this causing any break on existing traffic?

Reply:
no, this is already discussed with oracle who introduced this and they are fine with the change as there is no traffic `persistent_oauth2.0`